sources/HUBViewControllerImplementation.m

```diff
@@ -1314,6 +1337,105 @@ - (void)removeComponentWrapperFromLookupTables:(nullable HUBComponentWrapper *)c
     }
 }
 
+- (void)scrollToRootComponentAtIndex:(NSUInteger)componentIndex
+                      scrollPosition:(UICollectionViewScrollPosition)scrollPosition
+                            animated:(BOOL)animated
+                          completion:(void (^)())completionHandler
+{
+    NSUInteger const componentCount = (NSUInteger)[self.collectionView numberOfItemsInSection:0];
+
+    NSParameterAssert(componentIndex != NSNotFound);
```

Comment:
This is not needed as your next check will fail if the value is `NSNotFound` (since it's `INT_MAX`)